ERROR - 2025-03-11 15:56:14 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: YES) in C:\xampp\htdocs\codeigniter_4_rest_api\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\codeigniter_4_rest_api\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 C:\xampp\htdocs\codeigniter_4_rest_api\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\codeigniter_4_rest_api\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\codeigniter_4_rest_api\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\codeigniter_4_rest_api\app\Models\CommonModel.php(33): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codeigniter_4_rest_api\app\Controllers\Home.php(23): App\Models\CommonModel->selectRecord('apis')
#6 C:\xampp\htdocs\codeigniter_4_rest_api\system\CodeIgniter.php(933): App\Controllers\Home->index()
#7 C:\xampp\htdocs\codeigniter_4_rest_api\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\codeigniter_4_rest_api\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\codeigniter_4_rest_api\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\codeigniter_4_rest_api\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\codeigniter_4_rest_api\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: YES) in C:\xampp\htdocs\codeigniter_4_rest_api\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\codeigniter_4_rest_api\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\codeigniter_4_rest_api\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\codeigniter_4_rest_api\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\codeigniter_4_rest_api\app\Models\CommonModel.php(33): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\codeigniter_4_rest_api\app\Controllers\Home.php(23): App\Models\CommonModel->selectRecord('apis')
#5 C:\xampp\htdocs\codeigniter_4_rest_api\system\CodeIgniter.php(933): App\Controllers\Home->index()
#6 C:\xampp\htdocs\codeigniter_4_rest_api\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\codeigniter_4_rest_api\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\codeigniter_4_rest_api\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\codeigniter_4_rest_api\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\codeigniter_4_rest_api\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2025-03-11 15:56:14 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: YES)
[Method: GET, Route: /]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `apis`', [], false)
 3 APPPATH\Models\CommonModel.php(33): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Home.php(23): App\Models\CommonModel->selectRecord('apis')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-03-11 15:57:19 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: YES) in C:\xampp\htdocs\codeigniter_4_rest_api\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\codeigniter_4_rest_api\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', '', Object(SensitiveParameterValue), '', 3306, '', 0)
#1 C:\xampp\htdocs\codeigniter_4_rest_api\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\codeigniter_4_rest_api\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\codeigniter_4_rest_api\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\codeigniter_4_rest_api\app\Models\CommonModel.php(33): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codeigniter_4_rest_api\app\Controllers\Home.php(23): App\Models\CommonModel->selectRecord('apis')
#6 C:\xampp\htdocs\codeigniter_4_rest_api\system\CodeIgniter.php(933): App\Controllers\Home->index()
#7 C:\xampp\htdocs\codeigniter_4_rest_api\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp\htdocs\codeigniter_4_rest_api\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\codeigniter_4_rest_api\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\codeigniter_4_rest_api\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\codeigniter_4_rest_api\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: YES) in C:\xampp\htdocs\codeigniter_4_rest_api\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\codeigniter_4_rest_api\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\codeigniter_4_rest_api\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\codeigniter_4_rest_api\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\codeigniter_4_rest_api\app\Models\CommonModel.php(33): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\codeigniter_4_rest_api\app\Controllers\Home.php(23): App\Models\CommonModel->selectRecord('apis')
#5 C:\xampp\htdocs\codeigniter_4_rest_api\system\CodeIgniter.php(933): App\Controllers\Home->index()
#6 C:\xampp\htdocs\codeigniter_4_rest_api\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\codeigniter_4_rest_api\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\codeigniter_4_rest_api\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\codeigniter_4_rest_api\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 C:\xampp\htdocs\codeigniter_4_rest_api\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2025-03-11 15:57:19 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: YES)
[Method: GET, Route: /]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `apis`', [], false)
 3 APPPATH\Models\CommonModel.php(33): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Home.php(23): App\Models\CommonModel->selectRecord('apis')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
